### ISSUE TITLE
Trying to become trusted accessibility client on Windows

### ISSUE THREAD

**Issue by CodeF0x** [CodeF0x/violin#136]
**Describe the bug**
Violin tries to become a trusted accessibility client on Windows (and probably Linux, too) what fails, as this API exists only on Mac. There are no downsides for the user, and they won't even notice it, but it still produces an error in the console.

**To Reproduce**
Launch Violin on Windows and open DevTools.

**Expected behavior**
No error.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version v3.0
